src/components/GlobalPreferences/Notifications/NotificationModals.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Button, breakpoint, GU, Modal, textStyle } from '@aragon/ui'
+
+export function DeleteAccountConfirmationModal({
+  onConfirm,
+  onClose,
+  visible,
+}) {
+  return (
+    <Modal css="z-index: 2;" visible={visible} onClose={onClose}>
+      <h2
+        css={`
+          ${textStyle('title2')}
+        `}
+      >
+        Delete your account
+      </h2>
+      <p
+        css={`
+          margin-top: ${2 * GU}px;
+        `}
+      >
+        This action will unsubscribe you from all email notifications from all
+        organization' events.
+      </p>
+
+      <ModalControls>
+        <Button label="Cancel" onClick={onClose}>
+          Cancel
+        </Button>
+        <RemoveButton label="Unsubscribe" mode="negative" onClick={onConfirm}>
+          Unsubscribe
```

Comment:
Similar comment on button `wide`:

![image](https://user-images.githubusercontent.com/3072458/64104440-84d59100-cd74-11e9-9da7-db8742cb486e.png)

Reply:
Yes, all the buttons should be wide. Also, the spacing on the 'Create subscriptions' box should be the same as the boxes below:  

- When displayed on a viewport width < 768px => spacing is 16px on each side
- When displayed on a viewport width > 768px => spacing is 24px on each side 

I think Pierre has already made the changes at the component level.

Reply:
Resolved in https://github.com/aragon/aragon/pull/904/commits/a3205d1ee395e30242dd1d9cee4af55a23cf11ed